.gitattributes

```diff
@@ -0,0 +1,7 @@
+/test export-ignore
+/examples export-ignore
```

Comment:
since the `examples` dir includes actually runnable code, I figured it is best to not allow access to it in distribution packages.